controllers/controllers.js

```diff
@@ -1,7 +1,20 @@
 const { fetchTopicsData } = require("../models/models");
+const { fetchAPIsData } = require("../models/models");
 
```

Comment:
Remember to keep your code DRY - add fetchAPIsData to the object on line 1.

const { fetchTopicsData, fetchAPIsData } = require .... etc